# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../wine/ft_mulfix.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

X-Git-Url: http://source.winehq.org/git/wine.git/?a=blobdiff_plain;f=dlls%2Fgdi32%2Ffreetype.c;h=74e70ea27140e96f8e138660450eff3acd6589f5;hp=cb351db7693ef0bd763baf19331e0713c7f5480c;hb=de251e1c62651d8160b340a4fb9d0f9834a8e538;hpb=9c2ca354c29be96d0653123bff505304005f4f98

diff --git a/dlls/gdi32/freetype.c b/dlls/gdi32/freetype.c
index cb351db..74e70ea 100644
--- a/dlls/gdi32/freetype.c
+++ b/dlls/gdi32/freetype.c
@@ -163,7 +163,11 @@ MAKE_FUNCPTR(FT_Get_Sfnt_Table);
 MAKE_FUNCPTR(FT_Init_FreeType);
 MAKE_FUNCPTR(FT_Load_Glyph);
 MAKE_FUNCPTR(FT_Matrix_Multiply);
+#ifdef FT_MULFIX_INLINED
+#define pFT_MulFix FT_MULFIX_INLINED
+#else
 MAKE_FUNCPTR(FT_MulFix);
+#endif
 MAKE_FUNCPTR(FT_New_Face);
 MAKE_FUNCPTR(FT_New_Memory_Face);
 MAKE_FUNCPTR(FT_Outline_Get_Bitmap);
@@ -2434,7 +2438,9 @@ static BOOL init_freetype(void)
     LOAD_FUNCPTR(FT_Init_FreeType)
     LOAD_FUNCPTR(FT_Load_Glyph)
     LOAD_FUNCPTR(FT_Matrix_Multiply)
+#ifndef FT_MULFIX_INLINED
     LOAD_FUNCPTR(FT_MulFix)
+#endif
     LOAD_FUNCPTR(FT_New_Face)
     LOAD_FUNCPTR(FT_New_Memory_Face)
     LOAD_FUNCPTR(FT_Outline_Get_Bitmap)

